backend/src/BuiltinCliHost/Libs/Cli.fs

```diff
@@ -223,7 +224,12 @@ let fns : List<BuiltInFn> =
 
             try
               let parts = functionName.Split('.') |> List.ofArray
-              let name = NEList.ofList "PACKAGE" parts
+              let name =
+                NEList.ofListUnsafe
+                  "Can't call `Cli.executeFunction` with an empty function name"
+                  []
+                  parts
```

Comment:
this was broken, leading to `name` resulting in `["PACKAGE"; "PACKAGE"; "Darklang"; ...]`